students/kondel_tomasz/lesson_04_unit_testing/distance_test.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+import sys
+sys.path.insert(0, '../lesson_03_funtions/')
+from the_length_of_the_segment import distance  # noqa: E402
+
+
+def test_crash_with_TypeError():
```

Comment:
This is closer to ideal. `test_raises_TypeError_on_None` or `test_collatz_raises_TypeError_on_None`. The name of the function becomes more important as the number of tests grows to few hundreds. I guess `pytest` doesn't print the name of the module, so I include it... but maybe there is a way to turn it on.